test/physics/base/Physics.test.cc

```diff
@@ -229,6 +231,7 @@ TEST_F(PhysicsTrackViewHostTest, processes)
         const ParticleProcessId meow_ppid{1};
         EXPECT_EQ(ProcessId{3}, phys.process(hiss_ppid));
         EXPECT_EQ(ProcessId{4}, phys.process(meow_ppid));
+        EXPECT_TRUE(phys.has_at_rest());
```

Comment:
Perhaps we should change one of the mock particle types to have no at-rest process? Just to make sure we correctly handle that case.